common/cmake/devices.cmake

```diff
@@ -2134,7 +2137,7 @@ function(ADD_FPGA_TARGET)
   # Generate routing.
   # -------------------------------------------------------------------------
   add_custom_command(
-    OUTPUT ${OUT_ROUTE}
+    OUTPUT ${OUT_ROUTE} ${OUT_LOCAL}/route.log
```

Comment:
I see route.log being called multiple times. Should we replace it with a variable?

Reply:
Right, I added a variable.